packages/react-router-dom/package.json

```diff
@@ -34,7 +34,7 @@
     "prepublish": "node ./tools/build.js",
     "clean": "git clean -fdX .",
     "lint": "eslint modules",
-    "test": "karma start --single-run"
+    "test": "jest"
```

Comment:
Maybe, but I wouldn't consider this a "real" PR, I was just curious what would happen.

I think we could get away with using jest here as well, though. The only browser tests in `react-router-dom` are for `<BrowserRouter>` and `<HashRouter>` and all that those do is ensure that they create `history` objects. I'm not sure if it is just because Jest uses jsdom, but it appears that the `history` package can create browser and hash histories just fine, so those tests pass.

All of the other tests in `react-router-dom` use memory histories, so there isn't any value from running those tests in a browser.

The important thing would be to either use `babel-preset-env` or `eslint-plugin-compat` to catch any cross browser issues.